coreservices/invitation/src/CatenaX.NetworkServices.Invitation.Service/appsettings.json

```diff
@@ -115,18 +115,39 @@
       "emailVerified":true
     }
   },
-  "BasePortalAddress": "https://test.azurewebsites.net",
+  "Invitation": {
+    "Registration": {
+      "KeyCloakClientID": "",
```

Comment:
I cannot find where this value of KeyCloakClientID is actually being used? I can only see Portal.KeyCloakClientId being accessed.